pkg/cgroups/cgroups.go

```diff
@@ -178,6 +203,100 @@ func (subsys Subsystems) AddPID(name string, pid int) error {
 	return appendIntToFile(filepath.Join(subsys.CPU, name, "cgroup.procs"), pid)
 }
 
+func CreatePodCgroup(cgroupName string, config Config) error {
+	subsys, err := Find()
+	if err != nil {
+		return err
+	}
+	node, err := os.Hostname()
+	if err != nil {
+		return err
+	}
+	err = os.MkdirAll(filepath.Join(subsys.CPU, "p2", node), 0755)
+	if err != nil && !os.IsExist(err) {
+		return err
+	}
+	err = os.MkdirAll(filepath.Join(subsys.Memory, "p2", node), 0755)
+	if err != nil && !os.IsExist(err) {
+		return err
+	}
+	return create(cgroupName, "", config)
+}
+
+func CreateLaunchableCgroup(cgroupName, podID string, config Config) error {
```

Comment:
Can you create a similar public function that doesn't take a podID? I was confused when I found a potentially empty string being passed into this function on line 107 of p2-exec/main.go. In that file, use an if statement to call the correct version.

I'm less concerned about the private function taking an empty string.